report/assets/styles.css

```diff
@@ -931,16 +916,6 @@
   margin-bottom: calc(var(--audit-group-margin-bottom) / 2);
 }
 
-.lh-audit-group__header::before {
```

Comment:
This UX feature doesn't need to be PWA specific, but unless we have plans to start using it elsewhere I'm fine removing.